openpype/tools/publisher/publish_report_viewer/window.py

```diff
@@ -148,9 +201,66 @@ def update_file_content(self):
         self.content = content
         self.file_modified = file_modified
 
+    @staticmethod
+    def date_obj_to_timestamp(date_obj):
+        if hasattr(date_obj, "timestamp"):
+            return date_obj.timestamp()
+
+        # Python 2 support
+        epoch = datetime.datetime.fromtimestamp(0)
+        return (date_obj - epoch).total_seconds()
```

Comment:
Doesn't the report viewer always run under Py3 since it is a standalone tool?

Reply:
True, changed.